### ISSUE TITLE
API to be closed down

### ISSUE THREAD

**Issue by tangentfairy-zz** [CoderDojo/community-platform#364]
Need to lock down API except for coderdojo.com and changex.org (changeX need to be able to access this particular API endpoint: http://zentest.coderdojo.com:8000/api/1.0/dojos)

**Comment by wiizzard** [comment]
Is it possible to get public api access to the endpoints which do not require authentication (endpoints handled with "actHandler", not "actHandlerNeedsUser" etc.)? Currently this is not possible due to CORS.

We are especially interested in the https://zen.coderdojo.com/api/2.0/events/search endpoint to automatically keep the list of the upcoming Dojos updated. This would make things a lot easier as we manage our Dojos registration solely through the Zen platform. 

I don't see a security flaw coming up if access is only provided to the APIs requiring no authentication but I would like to hear your comments on that.

We have an implementation in progress at coderdojoka/coderdojoka.github.io#23. This may be also an starting point for other Dojos who like to implement similiar functionality.

**Comment by Wardormeur** [comment]
True, removing any CORS limitation on non-authenticated API calls could make sense. I need some insight from @DanielBrierton to validate the idea.
In the meantime, send an email to webteam{at}coderdojo.com, we're testing stuff :)
